src/Provider/Provider.Web/Controllers/DashboardController.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+using Esfa.Recruit.Provider.Web.Configuration.Routing;
+using Esfa.Recruit.Provider.Web.Orchestrators;
+using Esfa.Recruit.Provider.Web.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Esfa.Recruit.Provider.Web.Controllers
+{
+    [Route(RoutePrefixPaths.AccountRoutePath)]
+    public class DashboardController : Controller
+    {
+        private readonly DashboardOrchestrator _orchestrator;
+
+        public DashboardController(DashboardOrchestrator orchestrator)
+        {
+            _orchestrator = orchestrator;
+        }
+
+        [HttpGet("", Name = RouteNames.Dashboard_Index_Get)]
+        public IActionResult Dashboard([FromRoute]string ukprn)
```

Comment:
Should we maybe call this method index? or change the default route specified in the mvc route setup `Startup.Configure.cs`?, might be wrong in the recruit site too 😞

Reply:
We use attribute routing don't we??

Reply:
Have removed the default route pattern as we use attribute routing only currently.